packages/grafana-runtime/src/services/ScopesDashboardsService.ts

```diff
@@ -0,0 +1,259 @@
+import { isEqual } from 'lodash';
+import { BehaviorSubject } from 'rxjs';
+
+import { ScopeDashboardBinding, SuggestedDashboardsFoldersMap } from '@grafana/data';
+
+import { config } from '../config';
+
+import { getBackendSrv } from './backendSrv';
+
+export interface ScopesDashboardsServiceState {
```

Comment:
```suggestion
export interface ScopesDashboardContextValue {
```

I would stay away from "Service" name for the context value